<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorPlannerForm</class>
 <widget class="QWidget" name="colorPlannerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="selectCurrentLayerAsBaseButton">
       <property name="text">
        <string>Select Current Layer as Base Layer</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mask Selector</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>527</width>
        <height>339</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="buildMaskCompositorButton">
     <property name="text">
      <string>Build Parallel Mask Compositor</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
